charts/graphprotocol-indexer/templates/deployment.yaml

```diff
@@ -0,0 +1,119 @@
+apiVersion: apps/v1
```

Comment:
I think it's better to separate graphprotocol-indexer chart to two charts: graphprotocol-indexer-service, graphprotocol-indexer-agent, because they have many differences: env names, ports. IMHO `graphprotocol-indexer.envPrefix`, different images and if statements around `role` make this chart a little difficult to read and maintain.

Reply:
Separated charts